packages/cli/lib/services/sdk.ts

```diff
@@ -150,6 +150,7 @@ export class NangoSyncCLI extends NangoSyncBase {
         if (hasErrors.length > 0) {
             this.log('Invalid record payload. Use `--validation` option to see the details', { level: 'warn' });
             if (this.runnerFlags?.validateSyncRecords) {
+                console.log(JSON.stringify(hasErrors, null, 2));
```

Comment:
```suggestion
```

Reply:
Whoops, was from my other debugging with the issue reported https://nangohq.slack.com/archives/C07216H4BRT/p1742930735607089

Will remove this in a later PR

Reply:
Removed with https://github.com/NangoHQ/nango/pull/3775